components/file_attachment_list/file_attachment_list.jsx

```diff
@@ -79,6 +79,25 @@ export default class FileAttachmentList extends React.PureComponent {
             );
         }
 
+        if (fileInfos && fileInfos.length === 2) {
+            const fileType = getFileType(fileInfos[1].extension);
```

Comment:
Here you are checking type of only one file out of the two. This is fine for the first draft But this needs to be changed for the final version.